plugins/communicators/ssh/communicator.rb

```diff
@@ -21,6 +21,7 @@ module CommunicatorSSH
     class Communicator < Vagrant.plugin("2", :communicator)
       PTY_DELIM_START = "bccbb768c119429488cfd109aacea6b5-pty"
       PTY_DELIM_END = "bccbb768c119429488cfd109aacea6b5-pty"
+      CMD_GARBAGE_MARKER = SecureRandom.uuid.freeze
```

Comment:
I agree. I recommended the UUID, but instead of generating it on every invocation I'd grab one from an irb shell or something and paste it in here. For additional help with recognizability I usually prefix it with "vagrant ssh marker: " or something.